camayoc/tests/qcs/api/v1/utils.py

```diff
@@ -4,21 +4,14 @@
 import pprint
 import time
 
-from camayoc import config
```

Comment:
Were the deletions here due to dead code? Just trying to understand how it fits in with this PR.

Reply:
It came up because I was trying to figure out if any code still used the "sources" section under the "qcs" section in the config file. I figured out that I had actuall removed all use of this section from the API and had failed to remove this dead code. I was hoping to get rid of the section all together from the config file, but it turns out it is still used in some CLI tests. Essentially, this section of the config file is on the chopping block and will hopefully go away soon. I wanted to remove this dead code because I didn't want to document it or confuse people.

Reply:
What about squashing all commits but leaving these changes into a separated commits? I know it is related but I think the changes in this file and the changes on the other files can have a logical separation